hawc/apps/epimeta/api.py

```diff
@@ -1,3 +1,4 @@
+from rest_framework import serializers as ser
 from rest_framework import viewsets
```

Comment:
change line 2 to `from rest_framework import serializers, viewsets` instead of another import, both here and in other files